### PR TITLE
Cleanup harmony scripts #513

### PR DESCRIPTION
**Pull request by wardpeet**
I think this is a pretty nice cleanup of the harmony scripts #513

### CODE REVIEW

**Comment by wardpeet** [comment]
Looks like I need some default behaviour for harmony config.

**Comment by wardpeet** [comment]
Fixed the issue but looks like travis is not triggered.

**Comment by brendankenny** [comment]
restarted travis but looks like node4 isn't being run with `--harmony`: 

```
> lighthouse@1.0.5 cli-unit /home/travis/build/GoogleChrome/lighthouse
> mocha $(npm config get harmony) $(find lighthouse-cli/test -name '*.js') --timeout 60000
/home/travis/build/GoogleChrome/lighthouse/lighthouse-core/lib/log.js:33
  const args = [...logargs].slice(1);
                ^^^
SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at ...

npm ERR! Linux 3.19.0-30-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v4.3.2/bin/node" "/home/travis/.nvm/versions/node/v4.3.2/bin/npm" "run" "cli-unit"
npm ERR! node v4.3.2
npm ERR! npm  v2.14.12
npm ERR! code ELIFECYCLE
npm ERR! lighthouse@1.0.5 cli-unit: `mocha $(npm config get harmony) $(find lighthouse-cli/test -name '*.js') --timeout 60000`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the lighthouse@1.0.5 cli-unit script 'mocha $(npm config get harmony) $(find lighthouse-cli/test -name '*.js') --timeout 60000'.
npm ERR! This is most likely a problem with the lighthouse package,
```

**Comment by wardpeet** [comment]
Fixed :) Perhaps there is a better way than replacing undefined with an empty string

File: .travis.yml [hunk 1]
```diff
@@ -25,10 +19,11 @@ before_script:
   - start-stop-daemon --start --background --exec $(pwd)/lighthouse-core/scripts/launch-chrome.sh
   - sleep 5
 script:
+  - if [[ $(node -v) =~ ^v4.* ]]; then npm config set harmony=--harmony; else npm config rm harmony; fi
```
Comment (paulirish) [inline .travis.yml]: if instead of setting an npm variable we just `export __node_harmony=true` then no npmrc or `sed /undefined//` required. wdyt?

**Comment by wardpeet** [comment]
i could live with `export __node_harmony=--harmony` than i could replace the `npm config get` with just `$(echo $__node_harmony);`

**Comment by paulirish** [comment]
> i could live with export __node_harmony=--harmony than i could replace the npm config get with just $(echo $__node_harmony);

Cool. Yeah I am into it too. :)

**Comment by wardpeet** [comment]
Updated

**Pull request merged**